app/policies/grant_submission/submission_policy.rb

```diff
@@ -67,8 +67,8 @@ def grant
     @grant
   end
 
-  def current_user_is_applicant?
-    submission.applicant == user
+  def current_user_is_submitter?
+    submission.submitter == user
```

Comment:
these are going to need to look at applicant

Reply:
i expect that `app/policies/grant_submissoin/review_policy.rb` will need to be updated

Reply:
the blocking of the reviewer happens in the review model and that was changed to reflect applicants. There is nothing in the review policy about submitters.